src/Domain/Infra/Bundle/ServiceConfigHelper.php

```diff
@@ -33,6 +33,8 @@ public static function configureEntityFactory(ContainerBuilder $container, array
             ->setArgument('$idMapping', $idMapping)
             ->setArgument('$factory', new Reference('msgphp.entity_factory'))
             ->addTag('msgphp.entity_factory');
+
+        $container->register(CommandBusInterface::class, CommandBusInterface::class);
```

Comment:
i guess a lot of thinks break if simplebus is not installed :) it feels to wrong to register it here, where we are registering the entity factory. Besides registering it as a interface wont work in practice anyway right?

in general i like to make everything a no-op on missing requirements. Thus cleanup everything that requires command bus. Or wait a bit till SF message component is shipped so we have a library implementation to use / depend on.